onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -3606,30 +3611,34 @@ def aten_from_file(
 
 @torch_op("aten::full", trace_only=True)
 def aten_full(
-    size: INT64,
+    size: Union[INT64, INT32],
     fill_value: FLOAT,
```

Comment:
Should fill_value be a TensorType instead? Given that fill_value can be an int or a bool as well?

Reply:
Done